glam/api/management/commands/import_glean_aggs.py

```diff
@@ -27,18 +25,29 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser):
         parser.add_argument(
-            "product", help="The Glean product we are importing data for.",
+            "product", help="The Glean product we are importing data for."
+        )
+        parser.add_argument(
+            "--bucket",
+            help="The bucket location for the exported aggregates",
+            default=GCS_BUCKET,
         )
 
-    def handle(self, product, *args, **options):
+    def handle(self, product, bucket, *args, **options):
 
         csv_prefix = f"glam-extract-{product}"
         model = apps.get_model(PRODUCT_MODEL_MAP[product])
 
         self.gcs_client = storage.Client()
 
-        blobs = self.gcs_client.list_blobs(GCS_BUCKET)
-        blobs = list(filter(lambda b: b.name.startswith(csv_prefix), blobs))
+        blobs = self.gcs_client.list_blobs(bucket)
+        blobs = list(
+            filter(
+                lambda b: b.name.startswith(csv_prefix)
+                and not b.name.endswith("counts.csv"),
```

Comment:
Following the naming convention in the glam bucket, this needs to avoid mixing up the probe aggregates with the user counts.